lib/functions/lintly.sh

```diff
@@ -61,20 +73,5 @@ function SupportsLintly() {
   # Pull in the vars #
   ####################
   LANGUAGE="${1}"
-
-  echo "----<<<<INVOKING SupportsLintly>>>>----"
-
-  if [[ -v LINTLY_SUPPORT_ARRAY["${LANGUAGE}"] ]]; then
-    echo "${LANGUAGE} found inside LINTLY_SUPPORT_ARRAY."
-    true
-  else
-    echo "${LANGUAGE} NOT found inside LINTLY_SUPPORT_ARRAY."
-    false
-  fi
+  [[ -v LINTLY_SUPPORT_ARRAY["${LANGUAGE}"] ]]
 }
```

Comment:
love the cleanup on this method

Reply:
Felt like I learned a lot about shell scripting within the past couple days (and probably more than I would've liked to!). Apparently there were some bad practices in the existing code in this repo too, according to some 'Allegedly Experienced Shell Scripters' -- not that I could tell whose opinion I should listen to over another, anyway.